pkgs/tools/admin/zbctl/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchurl, coreutils }:
+
+stdenv.mkDerivation rec {
```

Comment:
Could use `stdenvNoCC` instead to avoid pulling in the C compiler.

Reply:
Done